lib/modules/asset/AssetService.ts

```diff
@@ -348,21 +357,25 @@ export class AssetService {
       { refresh, source: true }
     );
 
-    await Promise.all(
-      replacedAssets.successes.map((asset) =>
-        this.assetHistoryService.add<AssetHistoryEventMetadata>(
-          engineId,
-          {
-            metadata: {
-              names: Object.keys(flattenObject(asset._source.metadata)).concat(
-                removedMetadata.map((name) => `-${name}`)
-              ),
-            },
-            name: "metadata",
+    const histories: AssetHistoryContent[] = replacedAssets.successes.map(
+      (asset) => ({
+        asset: asset._source as AssetContent,
+        event: {
+          metadata: {
+            names: Object.keys(flattenObject(asset._source.metadata)).concat(
+              removedMetadata.map((name) => `-${name}`)
```

Comment:
The `names` array is supposed to contains the metadata names without additional information. To know if the metadata was removed/modified/added then users need to compare with previous history document

Reply:
It seems to have been added recently by Florian, in order to fix [an issue](https://github.com/kuzzleio/kuzzle-device-manager/pull/287/files#diff-7afb8db4a98276c4e4429a694fb1610155bda81aa8a15cc17fdd093daaea4c13R356-R358)

Reply:
@fmauNeko Can you give us more insight on this please?

Reply:
I added it to clearly mark removed metadata. If that was wrong, feel free to revert it, or I can open a PR to remove that.

Reply:
The first goal was to be able to search for history document regarding a metadata to retrace modifications so for now it's better to keep the name only :+1: 

We could add more info with addition/deletion/modification if needed